core/DCache.hpp

```diff
@@ -91,6 +105,7 @@ namespace olympia {
         const bool l1_always_hit_;
         bool busy_;
         uint32_t cache_latency_;
+        uint64_t cache_line_size_;
         uint32_t max_mshr_entries_;
```

Comment:
These variables are uninitialized.

Reply:
I initialize them in the DCache constructor

Reply:
Ah, you're missing my point.  😄   You should _ALWAYS_ provide a default value when declaring a variable _just in case_ it's not or forgotten in the constructor.

If you do not want to provide a default value, then make them `const` which will _force_ you to provide a value in the constructor.  In fact -- please make the `const` now so you have them initialized to constant values.  
Thanks!

Reply:
Understood, I have made those changes now